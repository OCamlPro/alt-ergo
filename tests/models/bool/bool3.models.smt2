(set-option :produce-assignments true)
(set-option :produce-models true)
(set-logic ALL)
(declare-const x Bool)
(declare-const y Bool)
(assert (or (! (and x y) :named foo) (! (and (not x) (not y)) :named bar)))
(check-sat)
(get-model)
(get-assignment)