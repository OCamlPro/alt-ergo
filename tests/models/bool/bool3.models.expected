
unknown
(
  (define-fun x () Bool true)
  (define-fun y () Bool true)
)
((bar false)
 (foo true))

